src/ray/common/test_util.cc

```diff
@@ -23,11 +23,27 @@
 namespace ray {
 
 void RedisServiceManagerForTest::SetUpTestCase() {
-  auto seed = std::chrono::high_resolution_clock::now().time_since_epoch().count();
-  std::mt19937 gen(seed);
-  std::uniform_int_distribution<int> random_gen{2000, 7000};
-  // Use random port to avoid port conflicts between UTs.
-  REDIS_SERVER_PORT = random_gen(gen);
+  std::vector<int> ACTUAL_REDIS_SERVER_PORTS;
+  if (REDIS_SERVER_PORTS.empty()) {
```

Comment:
nit: use google code style for this local variable? e.g. `actual_redis_server_ports`

Reply:
ok